src/shared/forms/useForm.ts

```diff
@@ -83,8 +83,8 @@ export const useForm = (config: FormConfig): UseForm => {
     setState(newState);
   };
 
-  const submit = (e: FormSubmitEvent): boolean => {
-    e.preventDefault();
+  const submit = (e?: FormSubmitEvent): boolean => {
+    e && e.preventDefault();
 
```

Comment:
e && e.preventDefault() - it means if event is passed then preventDefault behaviour, am I right?

Reply:
Yes